pages/api_integration-guides/Getting_started_with_the_typescript_client.mdx

```diff
@@ -0,0 +1,15 @@
+import { Tab, Tabs } from "nextra-theme-docs";
+
+# Getting Started with the TypeScript Client
+
+## Quickstart guides
+
+There are multiple quickstart guides to get you up and running with the TypeScript client library for dYdX V4. 
+
+### Run examples from the v4-client-js repository
+- Clone and run the examples from the v4-client-js repository. See the following [guide](https://github.com/dydxprotocol/v4-clients/tree/main/v4-client-js#readme) to run TS client examples from the [Typescript client repository](https://github.com/dydxprotocol/v4-clients/tree/main/v4-client-js). 
+
+
+### Setup your own project
+- Install and use the v4-client-js code as a package (in your own software) for your own [example project](https://github.com/dydxfoundation/solutions-public/tree/main/v4-client-js/examples), which you can modify freely
+and iterate on for your API trading use cases.
```

Comment:
Nitpick, not blocking: suggest rephrasing to separate the directive (install and use...) and the example project. As written, "for your own example project" isn't quite right (although intuitive enough that I doubt it will cause confusion)

```suggestion
- Install and use the v4-client-js code as a package in your own software. You can modify this [example project](https://github.com/dydxfoundation/solutions-public/tree/main/v4-client-js/examples) to work for your API trading use cases.
```